wercker.yml

```diff
@@ -41,6 +41,15 @@ build:
           bin/cfmt
           git diff --exit-code
     - script:
+        name: gas
+        code: |
+          go get github.com/GoASTScanner/gas
```

Comment:
I think pinning to a particular commit hash of gas makes sense at least, especially as there seem to be pretty frequent (breaking) changes upstream